phoenicis-configuration/src/main/resources/Linux.properties

```diff
@@ -28,6 +28,7 @@ application.user.engines.wine               =           ${application.user.engin
 application.user.engines.wine.gecko         =           ${application.user.engines}/wine/gecko/
 application.user.engines.wine.mono          =           ${application.user.engines}/wine/mono/
 application.user.resources                  =           ${application.user.root}/resources/
+application.user.settings                   =           ${application.user.root}/config.properties
```

Comment:
Does it prevent the program from running is  ${application.user.root} does not exist?